client-ci-analysis/ci-testfailure-analyzer/readme.md

```diff
@@ -0,0 +1,24 @@
+﻿# ci-testfailure-analyzer
+
+Tools to help analyze CI builds for failed/flaky tests, it generates cvs report into output directory specified in parameter.
+
+## Setup following env vars for authentication
```

Comment:
nit: Consider adding how many days this will look back for.

If it's not deterministic, you can either add that or just ignore my comment :)